token_auth/auth/base.py

```diff
@@ -21,11 +20,7 @@ def __init__(self, request, **kwargs):
         self.settings = get_settings()
 
     def sso_url(self, target_url=None):
-        url = self.settings['sso_url']
-        if target_url:
-            url += '?{}'.format(urllib.urlencode({'url': target_url}))
-
-        return url
+        raise NotImplemented()
```

Comment:
the main reasoning was that I have no idea if that is a sensible default. All other sso methods (mainly openid and oauth) will not have this as a default. So it might be very booking specific.